prebuilts/etcd.toml

```diff
@@ -0,0 +1,29 @@
+id = "etcd"
+name = "ETCD"
+icon = "https://avatars.githubusercontent.com/u/41972792?s=200&v=4"
+description = "etcd is a distributed key-value store designed to securely store data across a cluster."
+
+[source]
+image = "bitnami/etcd:3.5.10"
+
+[[ports]]
+id = "client_requests"
+port = 2379
+type = "TCP"
+
+[[ports]]
+id = "peer_communication"
+port = 2380
+type = "TCP"
+
+[[volumes]]
+id = "data1"
+dir = "/var/lib/etcd"
+
+[[volumes]]
+id = "data2"
+dir = "/var/etcd"
```

Comment:
etcd uses `/var/lib/etcd` as default data storage location, it seems we do not need this volume

Reply:
（found in docs

Reply:
removed